WcaOnRails/config/initializers/doorkeeper.rb

```diff
@@ -89,8 +89,8 @@
   # Forces the usage of the HTTPS protocol in non-native redirect uris (enabled
   # by default in non-development environments). OAuth2 delegates security in
   # communication to the HTTPS protocol so it is wise to keep this enabled.
-  #
-  # force_ssl_in_redirect_uri !Rails.env.development?
+
+  force_ssl_in_redirect_uri Rails.env.production?
```

Comment:
This is not directly related to my changes, but allowed me to clean up the oauth request spec a bit (previously it was using `save!(validate: false)` to create non https redirect_uris.